### ISSUE TITLE
GraphQL : `seomatic` field can not be registered on solspace events

### ISSUE THREAD

**Issue by denisyilmaz** [solspace/craft3-calendar#97]
I'm using solspace calendar in combination with the [SEOmatic plugin](https://github.com/nystudio107/craft-seomatic). SEOmatic registeres its own `seomatic` field to all element types to be able to get the belonging meta data in templates and in GraphQL.

Unfortunately its not possible to query the `seomatic` field inside the solspace calendar events like this:

```gql
{
  solspace_calendar {
    event {
      id
      slug
      title
      url
      seomatic {
        metaTitleContainer
      }
    }
  }
}
```

The plugin owner @khalwat pointed me in the direction that the solspace calendar plugin is not registering a generic `CalendarEventInterface` which he would need to be able to register the field accordingly. More information can be found here: https://github.com/nystudio107/craft-seomatic/issues/1037#issuecomment-1031024488

Would it be possible to register `CalendarEventInterface`?

**To reproduce**
Steps to reproduce the behaviour:
1. Add Solspace Calendar Plugin and SEOmatic and add a default event
2. Go into the GraphQL Explorer inside Craft CMS CP
3. Query for:
```gql
{
  solspace_calendar {
    event {
      id
      slug
      title
      url
      seomatic {
        metaTitleContainer
      }
    }
  }
}
```
4. See error message:  
```json
{
  "errors": [
    {
      "message": "Cannot query field \"seomatic\" on type \"CalendarEventInterface\".",
      "extensions": {
        "category": "graphql"
      },
      "locations": [
        {
          "line": 8,
          "column": 7
        }
      ]
    }
  ]
}
```

**Expected behaviour**
I would expect the `seomatic` field being available to `CalendarEventInterface` elements the same way as for `EntryInterface` elements. 

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.4.21
- Calendar Version: 3.3.8
- SEOmatic version: 3.4.23
- Calendar Edition:
- Fresh Install or Upgrade: fresh

**Comment by gustavs-gutmanis** [comment]
Hi @denisyilmaz,
Thanks for reporting this.

Could you try the `dev-master` version of `Calendar` and see if the issue is solved for you?

**Comment by denisyilmaz** [comment]
lovely. works flawlessly!

**Comment by kjmartens** [comment]
This should now be corrected in 3.3.9. 🙂
